app/views/posts/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<main id="posts" class="page">
+<%= render partial: 'shared/nav_link' %>
+
+    <div class=" user-item">
+       <%= render partial: 'shared/user_item', locals: { user: @user, show_link: false  } %>
+    </div>
+    
+  <article class="post-article">
+    <div class ="post_div" >
+    <% if @user.posts.empty? %>
+      <p>No posts added yet.</p>
+    <% else %>
+      <% @user.posts.each do |post| %>
+        <div class="post-container-div ">
+          <h3 class="post-title"><%= link_to post.title, "posts/#{post.id}" , class: "link-custom-color"%></h3>
+          <div class="post-details">
+            <p class="details"><%= truncate(post.text, length: 100) %></p>
+          </div>
+          
+          <div class="comment-like-container post-meta">
+            <p class="coment-likes-data ">
+              Comment: <%= post.comments_counter || 0 %>, Likes: <%= post.likes_counter || 0 %>
+            </p>
+          </div>
+          <% if can? :destroy, post %>
+  <%= button_to "Delete", user_post_path(user_id: post.author_id, id: post.id), method: :delete, data: { confirm: "Are you sure you want to delete this post?" }, class: "delete-btn" %>
```

Comment:
- [ Required ] I think it would be nice if you make it possible for a user with the `role` of `admin` to be able to `delete` any post and any `comment` on this page. Currently, when I log in as a user with the role of `admin` the `delete` button is not shown which makes it impossible for me to delete any post on the page. 

## Screenshot when I log in as admin
> There is no delete button on the posts index page when I login as a user with the `role` of `admin`. Please note that a user with the role of `admin` should be able to delete any `post` or `comment` on the blog application.

![Screenshot from 2023-09-07 23-04-24](https://github.com/Kel-nana/Blog-app/assets/16392046/5a4015b9-91b5-41e7-ab52-981aa2d3cac6)